planner/core/exhaust_physical_plans.go

```diff
@@ -2752,11 +2752,37 @@ func (la *LogicalAggregation) exhaustPhysicalPlans(prop *property.PhysicalProper
 }
 
 func (p *LogicalSelection) exhaustPhysicalPlans(prop *property.PhysicalProperty) ([]PhysicalPlan, bool, error) {
+	newProps := make([]*property.PhysicalProperty, 0, 3)
```

Comment:
Use 2 as the capacity of `newProps` is enough.

Reply:
+1. Updated.